src/iface/ethernet.rs

```diff
@@ -56,27 +56,29 @@ pub struct Interface<'b, 'c, DeviceT: for<'d> Device<'d>> {
 /// methods on the `Interface` in this time (since its `device` field is borrowed
 /// exclusively). However, it is still possible to call methods on its `inner` field.
 struct InterfaceInner<'b, 'c> {
-    neighbor_cache:         NeighborCache<'b>,
-    ethernet_addr:          EthernetAddress,
-    ip_addrs:               ManagedSlice<'c, IpCidr>,
+    neighbor_cache: NeighborCache<'b>,
```

Comment:
this might be a major headache - rustmt doesn't seem to work well for struct alignment, even though there is an [option](https://github.com/rust-lang-nursery/rustfmt/blob/master/Configurations.md#struct_field_align_threshold) for it. 

I guess this is the price to pay for having the formatter.